react/Carousel.tsx

```diff
@@ -310,14 +315,16 @@ export default class Carousel extends Component<Props, State> {
         </Slider>
         {showDots && (
           <Dots
+            loop
```

Comment:
I think the slider should handle the loop and pass through the dots. loop don't need to be part of Dots api

Reply:
It is needed in my implementation because the Dots component needs to know how many children the Slider component have, since the implementation of loop make clones of the slides in order to make the effect, another solution that I wanted to implement was getting the ref of Slider and pass it to Dots, but I couldn't do it because the server always puts some HOCs that wraps Slider.

Reply:
Another solution would be render the dots inside the Slider component, it would be a breaking change in the slider app, but not here, but it would make the Slider component even bigger both for usage and the code